tests/xharness/Jenkins.cs

```diff
@@ -1694,7 +1695,7 @@ function toggleAll (show)
 				var headerColor = "black";
 				if (unfinishedTests.Any ()) {
 					; // default
-				} else if (failedTests.Any ()) {
+				} else if (failedTests.Any () || skippedTests.Any ()) {
```

Comment:
might want to pick another colour (like orange) since the point of the PR is to separate failures from unknowns

Reply:
Fair enough.